src/main/java/uk/gov/register/filters/ContentSecurityPolicyFilter.java

```diff
@@ -8,10 +8,15 @@
 import javax.ws.rs.ext.Provider;
 import java.io.IOException;
 
+// the script digests are for the two inline scripts in govuk_template.gem:govuk_template.html.erb
+// if the scripts in that file change, or more are added, use a command similar to
+// this to generate the digests:
+// `echo "'sha256-"$(echo -n "inline javascript text" | openssl dgst -sha256 -binary | openssl enc -base64)"'"`
+
 @Provider
 public class ContentSecurityPolicyFilter implements ContainerResponseFilter {
     @Override
     public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
-        responseContext.getHeaders().add(HttpHeaders.CONTENT_SECURITY_POLICY, "default-src 'self' www.google-analytics.com");
+        responseContext.getHeaders().add(HttpHeaders.CONTENT_SECURITY_POLICY, "default-src 'self' www.google-analytics.com; style-src 'self' 'unsafe-inline'; object-src 'none'; font-src data:; img-src 'self'; script-src 'self' 'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=' 'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=' 'unsafe-inline';");
```

Comment:
I'm not sure you need the `style-src 'unsafe-inline'` since you never use an inline `<style>` tag? or does one get pulled in by govuk_template?